opta/commands/push.py

```diff
@@ -102,7 +129,10 @@ def push_to_docker(
         check=True,
     )
     nice_run(["docker", "tag", local_image, remote_image_name], check=True)
-    nice_run(["docker", "push", remote_image_name], check=True)
+    push_record = nice_run(
```

Comment:
my only question is what happens during the docker push command now? Right now we can see the docker push "working" in the cli with those nifty "layer upload progress bars". Are those gone now?

Reply:
It still prints those elements, but I am checking the Digest based on the Image tag that is being uploaded.